byron/ledger/executable-spec/test/Generator.hs

```diff
@@ -0,0 +1,109 @@
+module Generator where
+
+import           Data.Bits (shiftL)
+import           Data.Word (Word64)
+import           Hedgehog
+import qualified Hedgehog.Gen    as Gen
+import qualified Hedgehog.Range  as Range
+import           Ledger.Core (Slot(..), SlotCount(..), BlockCount(..))
+import           Ledger.GlobalParams (k)
+import           Ledger.Update (ProtVer(..), PParams(..))
+import           Numeric.Natural (Natural)
+
+-- | Generates a slot within the given bound
```

Comment:
Maybe move this to `Ledger.Core.Generator`?

Reply:
So we might want to have these generators in the `src` directory so that they can be used in the conformance testing at `cardano-ledger`.

Reply:
Done!